File: front/src/pages/imc/Imc.ts
import { Aluno } from '../Aluno';

export interface Imc {
    id : number;
    resultimc : Float32Array;
    altura : Float32Array;
    peso : Float32Array;
    classificacao : string;
    datacriacao : string;
    aluno : Aluno;
    alunoid : number;
}

File: API/Models/IMC.cs
using System;
namespace API.Models
{
    public class Imc
    {
        public int Id { get; set; }
        public float ResultImc { get; set; }
        
        public float Altura { get; set; }

        public float Peso { get; set; }

        public string? Classificacao { get; set; }

        public string? DataCriacao { get; set; }

        public Aluno? Aluno { get; set; }

        public int AlunoId { get; set; }
    }
}